recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
+  sha256: 8ec457f1d52894619329650c58e2597b168e3312a1129d7863e23abad7d54e9e
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - librsvg              # [not win]
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+    - librsvg  # [not win]
+
+test:
+  commands:
+    - $R -e "library('rsvg')"  # [not win]
+    - "\"%R%\" -e \"library('rsvg')\""  # [win]
+
+about:
+  home: https://github.com/jeroen/rsvg#readme
+  license: MIT
+  summary: Renders vector-based svg images into high-quality custom-size bitmap arrays using
+    'librsvg2'. The resulting bitmap can be written to e.g. png, jpeg or webp format.
+    In addition, the package can convert images directly to various formats such as
+    pdf or postscript.
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
We don't typically include this file in the conda package because it is incomplete.

```
$ cat rsvg/LICENSE 
YEAR: 2016
COPYRIGHT HOLDER: Jeroen Ooms
```

Do you know about the [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper) script? It'll handle this case and some of the other formatting issues for you (e.g. adding more maintainers).

Reply:
Argh, my bad!